client/components/widget/widget-config-directive.html

```diff
@@ -34,11 +57,23 @@
          target="_blank">Showdown</a>)
     </div>
     <div class="pk-sidebar-item-value">
-      <textarea class="form-control container-footer-text"
+      <textarea class="form-control widget-footer-text"
           ng-model="ngModel.footer_text" ng-disabled="!ngModel"></textarea>
     </div>
   </div>
 
+  <div class="pk-sidebar-item"
+      ng-show="ngModel.type">
+    <div class="pk-sidebar-item-value">
+      <md-checkbox class="widget-scroll-overflow" ng-model="ngModel.scroll_overflow">
```

Comment:
As discussed, I think it would be cleaner to move the overflow checkbox to the container properties, controlling if the widget height is an exact (maximum) height or a stretchable minimum height.